new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -183,6 +189,10 @@ public static void initAll() {
                 condition.getRemainingTimeInMS()))
         .until(operatorIsRunning(opNamespace));
 
+    JsonObject dockerConfigJsonObject = getDockerConfigJson(
```

Comment:
This doesn't work for Jenkins runs. You have to move the code which gets the "REPO" env var into the repo* variables before calling this. Otherwise this will create secret with dummy values which won't work on Jenkins.

Reply:
Fixing it.